pysqa/utils/basic.py

```diff
@@ -422,7 +422,9 @@ def _execute_command(
                 universal_newlines=True,
                 shell=not isinstance(commands, list),
             )
-        except subprocess.CalledProcessError:
+        except subprocess.CalledProcessError as e:
+            with open(os.path.join(working_directory, "pysqa.err"), "w") as f:
```

Comment:
Should it be a write or an append? And should we name the file `pysqa.log` instead?

Reply:
My assumption was that in each directory `pysqa` only submits one job. If that's not true we should probably use propper `logging`. I have no opinion on the filename though, I can change that.